UI/window-basic-main.hpp

```diff
@@ -1247,9 +1255,45 @@ public slots:
 	void UpdateContextBarDeferred(bool force = false);
 	void UpdateContextBarVisibility();
 
+signals:
+	/* Streaming signals */
+	void StreamingStarting(bool broadcastAutoStart);
+	void StreamingStarted(bool withDelay = false);
+	void StreamingStopping();
+	void StreamingStopped(bool withDelay = false);
+
+	/* Broadcast Flow signals */
+	void BroadcastFlowEnabled(bool enabled);
+	void BroadcastStreamReady(bool ready);
+	void BroadcastStreamActive();
+	void BroadcastStreamStarted(bool autoStop);
+
+	/* Recording signals */
+	void RecordingStarted(bool pausable = false);
+	void RecordingPaused();
+	void RecordingUnpaused();
+	void RecordingStopping();
+	void RecordingStopped();
+
+	/* Replay Buffer signals */
+	void ReplayBufEnabled(bool enabled);
+	void ReplayBufStarted();
+	void ReplayBufStopping();
+	void ReplayBufStopped();
```

Comment:
```suggestion
	void ReplayBufferEnabled(bool enabled);
	void ReplayBufferStarted();
	void ReplayBufferStopping();
	void ReplayBufferStopped();
```
We can just use the full name like in `UI/basic-controls.hpp`.

Reply:
We can't `ReplayBufferStopping` is a function in `OBSBasic`, so `ReplayBuf` was used for signals to avoid confusion with the existing function.

Reply:
Unfortunate.